model/src/main/resources/defaults.yml

```diff
@@ -0,0 +1,358 @@
+# Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+# A file with default configuration options used by ORT. These options can be overridden in custom configuration files.
+
+ort:
+  variablesToPropagate:
+    # OS-related variables or basic tools
+    - ANDROID_HOME
+    - HOME
+    - HTTP_PROXY
+    - http_proxy
+    - HTTPS_PROXY
+    - https_proxy
+    - JAVA_HOME
+    - JAVA_OPTS
+    - known_hosts
+    - LANG
+    - LS_COLORS
+    - netrc
+    - netrcPath
+    - NO_PROXY
+    - no_proxy
+    - NODE_PATH
+    - NODE_EXTRA_CA_CERTS
+    - PREFIX
+    - TERM
```

Comment:
We should also propagate `PATH` and `PWD`.

Reply:
`PATH` really seems to be required. Lots of tests were failing because tools could not be found.